pkgs/development/php-packages/grumphp/default.nix

```diff
@@ -1,32 +1,27 @@
-{ mkDerivation, fetchurl, makeWrapper, lib, php }:
+{ buildPHPproject, fetchFromGitHub, lib }:
 
-mkDerivation (finalAttrs: {
+buildPHPproject rec {
```

Comment:
New derivation factories should support the [`finalAttrs` pattern](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes).

Reply:
This is now done.

Reply:
Does not look like that is the case.

Reply:
Sorry I pushed an erroneous commit this morning.

Reply:
Job done.